segmentstore/server/src/main/java/io/pravega/segmentstore/server/DirectSegmentAccess.java

```diff
@@ -12,7 +12,6 @@
 import io.pravega.segmentstore.contracts.AttributeUpdate;
 import io.pravega.segmentstore.contracts.ReadResult;
 import io.pravega.segmentstore.contracts.SegmentProperties;
-import io.pravega.segmentstore.contracts.StreamSegmentStore;
```

Comment:
How did this change make it in here?

Reply:
Good question. Perhaps related to the merge. I can't undo it, because checkstyle should fail with an unused import... 
Which leads to the question how was it like that to begin with...